### PR TITLE
feat: [CDS-37444]: move time selection on top right corner on service dashboard

### PR DESCRIPTION
**Pull request by Sagar-Rajput7**
##### Summary:

> Moved time selection to the top right corner of the service dashboard page.
> Renamed service instances to active instance count
> Also added a test case

##### Jira Links:

https://harness.atlassian.net/browse/CDS-37444

##### Screenshots:

Before :
Screenshot 2022-05-16 at 10 42 01 PM

Now:

https://user-images.githubusercontent.com/98325173/168641625-3c2c59be-23ee-4e34-8df2-3ff25c1deff2.mov

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by Sagar-Rajput7** [comment]
retrigger jest

**Review by AshwinBhatkal-Harness — DISMISSED** [review]

**Review by bhavya-sinha — APPROVED** [review]

File: src/modules/75-cd/components/Services/Services.tsx [hunk 1]
```diff
@@ -31,24 +52,34 @@ export const Services: React.FC = () => {
         fetchDeploymentList
       }}
     >
-      <Page.Header title={getString('services')} breadcrumbs={<NGBreadcrumbs />} />
+      <Page.Header
+        title={getString('services')}
+        breadcrumbs={<NGBreadcrumbs />}
+        toolbar={
+          <>
```
Comment (bhavya-sinha) [inline src/modules/75-cd/components/Services/Services.tsx:59]: This is not needed

File: src/modules/75-cd/components/Services/Services.tsx [hunk 2]
```diff
@@ -31,24 +52,34 @@ export const Services: React.FC = () => {
...
-              {
-                id: 'dashboard',
-                title: 'Dashboard',
-                panel: <ServicesDashboardPage />
-              },
-              { id: 'manageServices', title: 'Manage Services', panel: <ServicesListPage /> }
-            ]}
-          />
-        </div>
+        <DeploymentsTimeRangeContext.Provider value={{ timeRange, setTimeRange }}>
```
Comment (bhavya-sinha) [inline src/modules/75-cd/components/Services/Services.tsx:67]: Hope we are using the context for `ServicesDashboardPage`

File: src/modules/75-cd/components/Services/Services.tsx [hunk 3]
```diff
@@ -31,24 +52,34 @@ export const Services: React.FC = () => {
...
+          <div className={css.tabs}>
+            <Tabs
+              id={'serviceLandingPageTabs'}
+              defaultSelectedTabId={'dashboard'}
+              tabList={[
+                {
+                  id: 'dashboard',
+                  title: 'Dashboard',
```
Comment (bhavya-sinha) [inline src/modules/75-cd/components/Services/Services.tsx:75]: Why is title hardcoded. Better use from string file

File: src/modules/75-cd/components/Services/Services.tsx (hunk 2 above)
Comment (Sagar-Rajput7) [reply src/modules/75-cd/components/Services/Services.tsx:63]: yeah we use this on the service dashboard and service detail dashboard also, and even on CD Dashboard

File: src/modules/75-cd/components/Services/Services.tsx (hunk 3 above)
Comment (Sagar-Rajput7) [reply src/modules/75-cd/components/Services/Services.tsx]: resolved

File: src/modules/75-cd/components/Services/Services.tsx (hunk 1 above)
Comment (Sagar-Rajput7) [reply src/modules/75-cd/components/Services/Services.tsx]: resolved

**Review by bhavya-sinha — APPROVED** [review]

**Review by AshwinBhatkal-Harness — DISMISSED** [review]

**Comment by Sagar-Rajput7** [comment]
trigger cypress

**Review by PratyushGar — APPROVED** [review]

**Review by goyalsonali — APPROVED** [review]

**Review by kapilkumawat86 — APPROVED** [review]

**Pull request merged**